src/main/java/com/google/api/codegen/transformer/BundlingTransformer.java

```diff
@@ -75,10 +82,32 @@ public BundlingConfigView generateBundlingConfig(MethodTransformerContext contex
     bundlingConfigView.requestByteThreshold(bundlingConfig.getRequestByteThreshold());
     bundlingConfigView.requestByteLimit(bundlingConfig.getRequestByteLimit());
     bundlingConfigView.delayThresholdMillis(bundlingConfig.getDelayThresholdMillis());
+    bundlingConfigView.flowControlElementLimit(bundlingConfig.getFlowControlElementLimit());
+    bundlingConfigView.flowControlByteLimit(bundlingConfig.getFlowControlByteLimit());
+    bundlingConfigView.flowControlLimitExceededBehavior(
+        getFlowControlLimitBehavior(bundlingConfig.getFlowControlLimitExceededBehavior())
+            .toString());
 
     return bundlingConfigView.build();
   }
 
+  private static FlowControlLimitBehavior getFlowControlLimitBehavior(
+      FlowControlLimitExceededBehavior behavior) {
+    switch (behavior) {
+      case BLOCK:
+        return FlowControlLimitBehavior.Block;
```

Comment:
It seems a little silly to use an enum just to convert another constant/enum into a String. Maybe just return a String?